python-opendata/requirements.txt

```diff
@@ -0,0 +1,5 @@
+arrow==0.12.1
+ipython==6.4.0
```

Comment:
Is it really necessay

Reply:
No. As is pytest. See comment about Pipfile.

Reply:
You could make a `testing_requirements.txt` and `optional_requirements.txt`. It avoids to install thing without needing them for a minimal use.

Reply:
Yes I completely agree, I use it like that too. My `pip freeze` was just a first step away from Pipfile.